drivers/wifi/nrf700x/src/wifi_util.c

```diff
@@ -981,6 +1033,13 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		      nrf_wifi_util_dump_rpu_stats,
 		      1,
 		      1),
+	SHELL_CMD_ARG(rpu_lockup_recovery_stats,
+		      NULL,
+		      "Display RPU lockup recovery stats "
+		      "Parameters: umac or lmac or phy or all (default)",
```

Comment:
UMAC or LMAC or PHY

Reply:
Previously, in rpu_stats it's "umac or lmac or all stats" only.
We can change this to " lmac or all stats" only.